cookbooks/steinberger/doc/steinberger.tex

```diff
@@ -0,0 +1,157 @@
+Two different formats are currently supported: Perple\_X and HeFESTo. 
+The format needs to be selected in the input file, and each format has a specific header and needs to be structured in a specific way. The paragraph below explains how to structure a Perple\_X file. 
+This file format is the default, it is also the more flexible format and it is what is used in this cookbook. Since the only requirements for the format are the header and the order of some of the columns, 
+files created with other mineral physics software can also be converted to this format.  
+
+The Perple\_X header contains the following in the first 13 lines: 
+The Perple\_X version, the name of the data table, the dimensions of the data table (for example, for a table with one dimension being pressure, the other temperature, this would be 2), the variable in the first dimension (this either needs to be `T(K)' for temperature, or `P(bar)' for pressure), the minimum value of this variable, the increments this variable will be increased with in the table, and the number of different values of this variable the table contains, the second variable (again with minimum, increment and number of values), the number of material properties in the table, and finally, the names of the columns. The first two columns need to be the pressure and temperature (in any order). The other required column names are:
+`rho,kg/m3' (for the density), `alpha,1/K' (for the thermal expansivity), `cp,J/K/kg' (for the specific heat), `vp,km/s' (for the P-wave velocity), `vs,km/s' (for the S-wave velocity)), `h,J/kg' (for the specific enthalpy). 
+Optionally, the file can contain columns with the name `phase' (to read in the name of the dominant phase), and columns named `vol\_fraction\_' and the name of a phase after the second underscore (to read in volume fractions of different phases). 
+As an example, the header of the table used in this cookbook is given below:
+
+\begin{lstlisting}
+|6.6.6
+PYR-Ringwood88_2.tab                                                                                
+           2
+T(K)    
+   400.00360000000001     
+   19.999960000020000     
+         181
+P(bar)  
+   15001.334999999999     
+   5114.9322988556905     
+         262
+           8
+T(K)           P(bar)         rho,kg/m3      alpha,1/K      cp,J/K/kg      vp,km/s        vs,km/s        h,J/kg 
+\end{lstlisting}
+
+It is also useful to know that \aspect{} does not actually read in the values of the pressures and temperatures in the first two columns, but instead uses the minimum, increment, and number of values parameters given in the header, assuming a uniform step size. The first column is always assumed to be the inner loop (i.e., it needs to increase first while the second column stays constant). 
```

Comment:
I'm confused by the word "column" since the file above doesn't actually have rows/columns. Can you explain what "column" refers to here?